adding rum-cli-examples (#8984)
aws/aws-cli: awscli/examples/rum/delete-rum-metrics-destination.rst

**To deletes a destination for CloudWatch RUM extended metrics**

> The following ``delete-rum-metrics-destination`` deletes a destination for CloudWatch RUM extended metrics. If the command succeeds, no output is returned. ::

Reviewer: ```suggestion
The following ``delete-rum-metrics-destination`` example deletes a destination for CloudWatch RUM extended metrics. ::
```

> 	aws rum delete-rum-metrics-destination \

Reviewer: Same here. All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general

		--app-monitor-name AWSApp \
		--destination Evidently \
		--destination-arn arn:aws:evidently:us-east-1:123456789012:project/petfood/feature/petfood-upsell-text

> 

Reviewer: ```suggestion

This command produces no output.

```
